service-impl/cache-service-impl/simple-cache-service/src/main/java/uk/gov/gchq/palisade/cache/service/impl/SimpleCacheService.java

```diff
@@ -55,6 +64,11 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SimpleCacheService.class);
 
+    /**
+     * The maximum allowed time to live for entries that are marked as locally cacheable.
+     */
+    public static final Duration MAX_LOCAL_TTL = Duration.of(5, ChronoUnit.MINUTES);
```

Comment:
This needs to be configurable as part of the deployment

Reply:
Done